pom.xml

```diff
@@ -77,6 +78,12 @@
                     <argLine>
                         -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
                     </argLine>
+                    <properties>
+                        <property>
+                            <name>dataproviderthreadcount</name>
+                            <value>${thread.count}</value>
```

Comment:
in TestNG there is an option called `threadcount`, so the name `thread.count` for other option is confusing.
http://testng.org/doc/documentation-main.html#running-testng

Reply:
any suggestions on naming? `dataproviderthreadcount`? `threads`?

Reply:
are you sure that we should use dataproviderthreadcount not threadcount?
if so, we can say data.provider.thread.count

Reply:
pretty sure it is `dataproviderthreadcount`
https://cucumber.io/docs/guides/parallel-execution/#testng
> The default thread count of the dataprovider in parallel mode is 10. To change this the `dataproviderthreadcount` property needs to be added to the configuration section of the Surefire or Failsafe plugin in the POM.

Reply:
year, but as I see we parallelized by classes, not by methods. I'm not sure, is this correct when we use `dataproviderthreadcount` option, does it take any effect?

Reply:
is there a way to limit threads in current version?

Reply:
threadcount? :)

Reply:
nope
```
<properties>
   <property>
       <name>threadcount</name>
       <value>${thread.count}</value>
   </property>
</properties>
```
does not work in our case